examples/superminimal/arc.yml

```diff
@@ -0,0 +1,26 @@
+level_of_theory: 'b3lyp/6-31g(d,p)//b3lyp/6-31g(d,p)'
```

Comment:
if the opt and sp parts of `level_of_theory` are identical, it's enough to just specity
```python
level_of_theory: 'b3lyp/6-31g(d,p)'
```
The current state is also correct, decide which version you'd like to implement